src/gui/tray/UserModel.cpp

```diff
@@ -417,6 +444,36 @@ void User::slotAddError(const QString &folderAlias, const QString &message, Erro
     }
 }
 
+void User::slotAddErrorToGui(const QString &folderAlias, SyncFileItem::Status status, const QString &errorMessage, const QString &subject)
+{
+    const auto folderInstance = FolderMan::instance()->folder(folderAlias);
+    if (!folderInstance)
+        return;
+
+    if (folderInstance->accountState() == _account.data()) {
+        qCWarning(lcActivity) << "Item " << folderInstance->shortGuiLocalPath() << " retrieved resulted in " << errorMessage;
+
+        Activity activity;
+        activity._type = Activity::SyncFileItemType;
+        activity._status = status;
+        const auto currentDateTime = QDateTime::currentDateTime();
+        activity._dateTime = QDateTime::fromString(currentDateTime.toString(), Qt::ISODate);
+        activity._expireAtMsecs = currentDateTime.addMSecs(activityDefaultExpirationTimeMs).toMSecsSinceEpoch();
+        activity._subject = !subject.isEmpty() ? subject : folderInstance->shortGuiLocalPath();
+        activity._message = errorMessage;
+        activity._link = folderInstance->shortGuiLocalPath();
+        activity._accName = folderInstance->accountState()->account()->displayName();
+        activity._folder = folderAlias;
+
+        // add 'other errors' to activity list
+        _activityModel->addErrorToActivityList(activity);
+
+        if (!_expiredActivitiesCheckTimer.isActive()) {
+            _expiredActivitiesCheckTimer.start(expiredActivitiesCheckIntervalMs);
```

Comment:
so by default `QTimer` are repeating when `Timer` qml items are not
I was trying to find where you had made it to repeat

Reply:
> so by default `QTimer` are repeating when `Timer` qml items are not
> I was trying to find where you had made it to repeat

@mgallien 

when `void User::slotAddErrorToGui(const QString &folderAlias, SyncFileItem::Status status, const QString &errorMessage, const QString &subject)` is called and the error is added - the timer starts. When the last error has been removed from the list - it stops. It will start again if a new error is added. Or, do you mean something else with `Qtimer` are reapiting? 

What needs to be changed in QML? I thought I don't need to touch .qml files for this to work.